purchases/build.gradle

```diff
@@ -4,6 +4,17 @@ apply plugin: 'org.jetbrains.dokka'
 
 android {
     namespace 'com.revenuecat.purchases.api'
+
+    productFlavors {
+        integrationTest {
+            testApplicationId obtainTestApplicationId()
+            testBuildType obtainTestBuildType()
+            packagingOptions {
+                resources.excludes.add("META-INF/LICENSE.md")
+                resources.excludes.add("META-INF/LICENSE-notice.md")
```

Comment:
This is the main change from before. Before we were excluding `META-INF/*`. Now, we are excluding these files that were causing the duplicated files (due to Junit library in instrumentation tests). I tried to exclude this only on instrumentation tests but there was no easy way... And I think this should be safe enough.